.config/forest.dic

```diff
@@ -30,3 +32,4 @@ tipset/SM
 tipsetkey/S
 tuple
 validator/S
+stringified
```

Comment:
It would be probably best to use some of the tricks (`/D`? not sure) to add just `stringify` without precise tense

Reply:
going to be using @lemmih idea to just reword it with "pretty-print"